src/DotNetWorker/Context/FunctionContext.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.Functions.Worker.Context;
+using Microsoft.Azure.Functions.Worker.Logging;
+
+namespace Microsoft.Azure.Functions.Worker.Pipeline
```

Comment:
Should we remove "Pipeline" from this namespace?

Reply:
Will do.